lib/Mojo/JSON/Pointer.pm

```diff
@@ -11,6 +12,8 @@ sub new { @_ > 1 ? shift->SUPER::new(data => shift) : shift->SUPER::new }
 sub _pointer {
   my ($self, $contains, $pointer) = @_;
 
+  Mojo::Exception->throw(__PACKAGE__.": Invalid JSON Pointer '$pointer'. Pointer must start with '/' or be undefined") if ($pointer && $pointer =~ /^[^\/].+/);
```

Comment:
Why do you use `Mojo::Exception` here? The tests don't even use any of its features.